blog/_posts/2024-05-14-core-extension.md

```diff
@@ -0,0 +1,87 @@
+
+#### Native AKS experience
+* Add-ons migrated to core extensions: Some AKS add-ons, such as Container Insights, Azure Managed Prometheus, and App Monitoring, are transitioning to core extensions. Although the underlying technology shifts to extensions, the user experience remains unchanged. The CLI command to enable/disable migrated add-ons are the same as before migration.
+
+    ```bash
+    az aks create/update --resource-group <group> --name <cluster> --enable/disable <add-on>
+    ```
+* Regular extensions graduated to core extensions: Key extensions like Azure Backup are also being upgrading to core extensions for better integration with AKS.
+
+    CLI commands:
+    * Create/delete a regular extension:
+    ```bash
+    az k8s-extension create/delete --extension-type <type> --resource-group <group> --cluster-name <name> --cluster-type <clusterType> --name <extension name>
+    ```
+    * Create/delete a core extension:
+    ```bash
+    az aks extension create/delete --extension-type <type> --resource-group <group> --cluster-name <name> --name <core extension name>
+    ```
+
+#### Safer version management
+In principle, core extensions follow the same update rules as add-ons: patch versions can be upgraded within a Kubernetes minor version, while major/minor upgrades occur only with Kubernetes minor version updates to avoid introducing breaking changes in a Kubernetes version. Exceptions are allowed if the extension has proven testing and safeguards against unexpected breaking changes in new minor/major versions.
+
+#### Improved security, efficiency and reliability
+
+ExtensionManager is moved from user node pool to AKS Control Plane, which improved extension security, efficiency, and reliability.
+
+What Is ExtensionManager?
+
+ExtensionManager is a key component that handles lifecycle operations (create, upgrade, delete, reconcile) for Kubernetes extensions. It periodically polls for version updates on extension instances and reconciles extension configuration settings, ensuring that extensions are managed effectively. Previously running on customer worker nodes, it has now been moved to the AKS control plane.
+
+What Has Changed?
+
+Previously, ExtensionManager pods (extension-operator and extension-agent) were deployed in the kube-system namespace on worker nodes. These components are now part of the AKS control plane, so they are no longer visible to users.
+
+Example of previous deployment:
+```bash
+$ kubectl get pods -n kube-system
+NAME                                      READY   STATUS    RESTARTS   AGE
+coredns-558bd4d5db-7x5c8                  1/1     Running   0          5d
+extension-agent-abc123                    1/1     Running   0          3d
+extension-operator-def456                 1/1     Running   0          3d
+```
```

Comment:
Could consider reordering these sections. From what can user see before, what changed, and what are the benefits